frontend/src/components/RichEditor.js

```diff
@@ -0,0 +1,88 @@
+/* istanbul ignore file */
```

Comment:
Ah I remember adding that in since the component is hard to test. For example, the Editor component uses an html attribute `contentEditable` which the react testing library does not have support for it. With that, there's no easy way to test that inputs and updates take effect. So I couldn't raise the test coverage inside that file b/c of that.
I agree that I don't like commenting it out but wasn't sure how I could solve the issue.

Reply:
Hmm.

Could there maybe be a test that at least verifies that the component renders, even without digging into trying to use it? If adding that test though drops the code coverage too far I'm ok with continuing to ignore this file since there isn't a whole lot of actual behavior here.

If we continue to ignore it, please do add a comment up near that declaration to explain why it's being ignored, and if there's an issue filed in react testing library that we could check to see when `contentEditable` is supported a link to that issue. Something to make it easier to know why it's being ignored and what might need to change to get it tested properly.